cmd/nfd-master/main.go

```diff
@@ -125,6 +125,9 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 	flagset.StringVar(&args.Options, "options", "",
 		"Specify config options from command line. Config options are specified "+
 			"in the same format as in the config file (i.e. json or yaml). These options")
+	flagset.DurationVar(&args.ResyncPeriod, "resync-period", 1,
```

Comment:
if you specify `1` as default value it'll means that `ResyncPeriod` is set to 1 nanosecond, which is not what we want.  If you want to specify an hour you should pass `time.Hour`.

Reply:
Yeah I know, it's actually WIP, I'll ping you once I finish.